### ISSUE TITLE
Query a list of DOIs

### ISSUE THREAD

**Issue by rossmounce** [lazappi/doilinker#1]
Bit of a feature request, not a bug per se

I would like to be able to submit a (large) list of DOIs not just one DOI.

e.g.

```
xvz <- search_doi_links(c("10.1101/2020.11.13.380444","10.1101/2021.01.19.427365","10.1101/2021.03.08.434485"))
```

or some such. Is that possible? I don't see an example in the documentation

**Comment by lazappi** [comment]
Yes, this would be good functionality to have (not sure why I didn't do that originally).

For now a simple workaround is probably to use `lapply()` (or another looping/mapping function):

```r
dois <- c("doi1", "doi2", ...)

matches <- lapply(dois, search_doi_links)
```

**Comment by rossmounce** [comment]
thx
